docs-devsite/vertexai.generationconfig.md

```diff
@@ -66,14 +66,24 @@ presencePenalty?: number;
 
 ## GenerationConfig.responseMimeType
 
-Output response mimetype of the generated candidate text. Supported mimetypes are `text/plain` (default, text output) and `application/json` (JSON response in the candidates). The model needs to be prompted to output the appropriate response type, otherwise the behavior is undefined. This is a preview feature.
+Output response MIME type of the generated candidate text. Supported MIME types are `text/plain` (default, text output) and `application/json` (JSON response in the candidates).
 
 <b>Signature:</b>
 
 ```typescript
 responseMimeType?: string;
 ```
 
+## GenerationConfig.responseSchema
+
+Output response schema of the generated candidate text. This value can be a class generated with a [Schema](./vertexai-preview.schema.md#schema_class) static method like `Schema.string()` or `Schema.object()` or it can be a plain JS object matching the [SchemaRequest](./vertexai-preview.schemarequest.md#schemarequest_interface) interface. <br/>Note: This only applies when the specified `responseMIMEType` supports a schema; currently this is limited to `application/json`<!-- -->.
```

Comment:
Shouldn't `responseSchema` also support the MIME type of `text/x.enum`?

https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/control-generated-output#considerations

Reply:
I guess so. Added. I don't really know anything about it other than what's in that page so I just put the type in there and didn't add any explanation. I originally pulled this comment off the proto. https://source.corp.google.com/piper///depot/google3/google/cloud/aiplatform/master/content.proto;l=269?q=generationconfig&ss=piper%2FGoogle%2FPiper:google3%2Fgoogle%2Fcloud%2Faiplatform%2Fmaster%2F 

I'm not sure what I'm supposed to use for the source of truth.

Reply:
Maybe I'm missing it, but I'm not seeing `text/x.enum` listed in this .md file as a possible MIME type?

Reply:
I added it to `responseMimeType` but not `responseSchema`. It should be here now also.